dockerfiles/base/tests/Dockerfile

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
it may be "script-tests" or "unit-test" or only "bats" as we're using bats 
but BTW, do we require to build our own custom bats docker image ?  none of bats images of dockerhub is working ?
for launcher we were using
https://github.com/eclipse/che/blob/5.1.0/dockerfiles/launcher/launcher_funcs_tests.bats#L14